client/app/components/packages/pas-form/edit.js

```diff
@@ -64,19 +68,29 @@ export default class PasFormComponent extends Component {
     return this.submittableChanges.isValid;
   }
 
-  @action
-  async save() {
-    await this.saveableChanges.save();
-
-    this.args.onSave();
+  get isSavingOrSubmitting() {
+    return this.save.isRunning || this.submit.isRunning;
   }
 
-  @action
-  async submit() {
-    await this.submittableChanges.save();
+  @task(function* () {
+    yield this.saveableChanges.execute();
 
-    this.args.onSubmit();
-  }
+    yield this.args.onSave();
+
+    yield this.saveableChanges.rollback();
```

Comment:
So this resets the changeset to its defaults after the model has been save?

Reply:
Yep, from my understanding of reading the docs. Usually when we call save() it does three things:

1. moves the Changeset changes to the underlying model
2. save the underlying model (which generates network PATCH requests)
3. reset the Changeset to a clean state (so it no longer reports any changes or the fact it is dirty) 

Since we no longer call .save(), we have to do these three steps manually. rollback() takes care of step 3.  

This is where it mentions it in the docs
https://github.com/poteto/ember-changeset#execute
![image](https://user-images.githubusercontent.com/3311663/81746538-abf07080-9474-11ea-8444-ea7f07156b1e.png)
I haven't extensively dug into this and verified it though. Maybe I should take some time and log the changeset without rollback() and see if it is dirty still.

Reply:
I think one use case of execute() not clearing away the changes is that you can apply the changes to multiple records if desired

Reply:
Yep, I just verified it. rollback() does indeed clear away Changeset.changes:
![image](https://user-images.githubusercontent.com/3311663/81747050-739d6200-9475-11ea-99b8-ae1bce21dfe2.png)

![image](https://user-images.githubusercontent.com/3311663/81747036-6f714480-9475-11ea-82d8-76b822ae7b0b.png)